### PR TITLE
Feat/dashboard

### PR DESCRIPTION
**Pull request by jentz2909**

### CODE REVIEW

File: src/pages/addNewInstancePage/index.jsx [hunk 1]
```diff
@@ -1,3 +1,74 @@
...
+        Securitykey:${securitykey}
+        Accesskey:${accesskey}
+        `);
+        event.preventDefault();
+    }
+
+
+    return (
+        <div className="AddingInstanceForm">
+            <h2>Add New Instace</h2>
```
Comment (tensan8) [inline src/pages/addNewInstancePage/index.jsx:45]: Typo: Must be "Instance"

File: src/pages/addNewInstancePage/index.jsx [hunk 2]
```diff
@@ -1,3 +1,74 @@
...
+                <input
+                type="text"
+                required
+                />
+                <label>IP Address: </label>
+                <input
+                type="text"
+                required
+                />
+                <label>SecurityKey: </label>
```
Comment (tensan8) [inline src/pages/addNewInstancePage/index.jsx:57]: Add space between the words

```suggestion
                <label>Security Key: </label>
```

File: src/pages/addNewInstancePage/index.jsx [hunk 3]
```diff
@@ -1,3 +1,74 @@
...
+                <input
+                type="text"
+                required
+                />
+                <label>AccessKey: </label>
```
Comment (tensan8) [inline src/pages/addNewInstancePage/index.jsx:62]: Add space between the words

```suggestion
                <label>Access Key: </label>
```

**Review by tensan8 — CHANGES_REQUESTED** [review]
Some comments have been given for typos on the add new instance page. Please address it first then you can merge it. There are some other notes about the responsiveness of the website but this one will be good to go for now.

CC @jentz2909

File: src/pages/addNewInstancePage/index.jsx (hunk 1 above)
Comment (tensan8) [inline src/pages/addNewInstancePage/index.jsx:45]: Fix Typos

```suggestion
            <h2>Add New Instance</h2>
```

**Comment by jentz2909** [comment]
done

**Review by tensan8 — APPROVED** [review]

**Pull request merged**